src/docs/product/metrics/metrics-explorer-page.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Metrics Explorer Page"
+sidebar_order: 12
+description: "Learn about how you can visualise and explore your metrics in the Sentry UI."
+---
+
+<Include name="feature-stage-alpha-metrics.mdx" />
+
+Once you've started sending metrics to Sentry, you can begin to explore them in the product by navigating to the Metrics page.
```

Comment:
```suggestion
Once you've started sending metrics to Sentry, you can explore them in the product using the **Metrics** page. It might take a few seconds for the first datapoints to appear.
```